apps/ledger-live-mobile/src/families/cosmos/Delegations/index.tsx

```diff
@@ -87,7 +89,22 @@ function Delegations({ account }: Props) {
     cosmosResources &&
     cosmosResources.unbondings &&
     mapUnbondings(cosmosResources.unbondings, validators, unit);
-
+  const bridge = getAccountBridge(account, undefined);
+  const { transaction, status } = useBridgeTransaction(() => {
+    const t = bridge.createTransaction(mainAccount);
+    const { validatorSrcAddress } = { ...banner };
+    return {
+      account,
+      transaction: bridge.updateTransaction(t, {
+        mode: "redelegate",
+        validators: [],
+        sourceValidator: validatorSrcAddress,
+
+        /** @TODO remove this once the bridge handles it */
```

Comment:
@haammar-ledger any clue why this was needed and what we meant by "the bridge handles it"?

Reply:
I just had a look and I'm pretty sure this isn't required anymore.
I think "_the bridge handles it_" just refers to the LLC part of the integration, which at some point in the past probably didn't support transactions without a recipient, but I just checked and I don't see a place where this is required now.

@sarneijim My advice is to just remove this line and check your fix still works (check the transaction fees too).
The rest LGTM :+1: